media-libs/svt-av1/svt-av1-0.8.6.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+
+DESCRIPTION="Scalable Video Technology for AV1 (SVT-AV1 Encoder and Decoder)"
+HOMEPAGE="https://github.com/AOMediaCodec/SVT-AV1"
+
+if [[ ${PV} = 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/AOMediaCodec/SVT-AV1.git"
+else
+	SRC_URI="https://github.com/AOMediaCodec/SVT-AV1/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 -x86" # -x86: https://github.com/AOMediaCodec/SVT-AV1/issues/1231
+	S="${WORKDIR}/SVT-AV1-${PV}"
+fi
+
+LICENSE="AOM BSD BSD-2 MIT"
+SLOT="0/0.8.6"
```

Comment:
```
ET_DYN libSvtAv1Dec.so.0 /usr/lib64/libSvtAv1Dec.so.0.8.6 
```
So they don't have SONAME declared. Do you **know** upstream breaks ABI between releases? And maybe not **every** release, but 0.8 -> 0.9?

I'd leave this undefined until upstream declares it.
`SLOT="0"`

Then apart the license/patent review from ulm, rest looks good to me.

Reply:
I thought `SLOT=0` was good too, but the change was a suggestion from review. https://github.com/gentoo/gentoo/pull/17110#discussion_r483676643